config/locales/vendor_outage/en.yml

```diff
@@ -0,0 +1,17 @@
+---
+en:
+  vendor_outage:
+    get_updates_on_status_page: Get updates on our status page
+    idv_blocked:
+      generic: We are having technical difficulties on our end and cannot verify your
+        identity at this time. Please try again later.
+      unfortunately:
```

Comment:
Could it make sense to flatten and remove this grouping altogether? I don't know that "unfortunately" adds much context to what the grouping is meant to contain.

i.e. `vendor_outage.idv_blocked.with_sp`

Reply:
yeah, I hated that as I typed it. Done in a7ebd67